prometheus-metrics-core/src/main/java/io/prometheus/metrics/core/metrics/Histogram.java

```diff
@@ -481,8 +481,8 @@ private boolean maybeReset() {
       nativeZeroCount.reset();
       count.reset();
       sum.reset();
-      for (int i = 0; i < classicBuckets.length; i++) {
-        classicBuckets[i].reset();
+      for (LongAdder classicBucket : classicBuckets) {
```

Comment:
the performance is not the same - but I guess it's ok here

Reply:
My understanding is that on a modern JVM the performance difference should be negligible.